component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mobileconnect/MobileConnectAuthenticator.java

```diff
@@ -1174,6 +1218,44 @@ protected String getCallbackUrl(Map<String, String> authenticatorProperties) {
         return callbackURL;
     }
 
+    public boolean shouldPrefixClaimDialectUri() {
```

Comment:
Why do you want this method to be public ?

Reply:
this method reads the configs and gives out the value (like getClaimDialectUri).
So making it public is intended to ease the work in case of an inheritance from MobileConnectAuthenticator.